addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/NetworkBindingConstants.java

```diff
@@ -8,11 +8,12 @@
  */
 package org.openhab.binding.network;
 
-import java.util.Collections;
 import java.util.Set;
 
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 
+import com.google.common.collect.Sets;
```

Comment:
Sets.newHashSet is used in every binding I have seen so far. To be honest I would not know how to accomplish something like `Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Sets.newHashSet(DEV1, DEV2);` without this helper class in one line without writing an own helper that would need to be replicated in every binding. Is there any solution yet?